web/2-inputnamebadge/piratebadge.dart

```diff
@@ -4,8 +4,16 @@
 
 import 'dart:html';
 
-void  main() {
-  querySelector('#inputName').onInput.listen(updateBadge);
+main() {
```

Comment:
This seems like one of those cases where it's better to use the old syntax.

The new syntax is trickier, and it bloats the file by 8 lines.